docs/v0.6/en/user-guide/client-libraries/go.md

```diff
@@ -239,25 +325,47 @@ m.DB = db
 
 {template query-library-raw-sql%
 
+The following code declares a GORM object model:
+
+```go
+type CpuMetric struct {
+    Host        string    `gorm:"column:host;primaryKey"`
+    Ts          time.Time `gorm:"column:ts;primaryKey"`
+    CpuUser     float64   `gorm:"column:cpu_user"`
+    CpuSys      float64   `gorm:"column:cpu_sys"`
+}
+```
+
+If you are using the [ORM API](#orm-api) to insert data, you can declare the model with both GORM and Greptime tags.
+
 ```go
-type Monitor struct {
-	Host        string    `gorm:"column:host;primaryKey"`
-	Ts          time.Time `gorm:"column:ts;primaryKey"`
-	Cpu         float64   `gorm:"column:cpu"`
+type CpuMetric struct {
+    Host        string    `gorm:"column:host;primaryKey" greptime:"tag;column:host;type:string"`
+    Ts          time.Time `gorm:"column:ts;primaryKey"   greptime:"timestamp;column:ts;type:timestamp;precision:millisecond"`
+    CpuUser     float64   `gorm:"column:cpu_user"        greptime:"field;column:cpu_user;type:float64"`
+    CpuSys      float64   `gorm:"column:cpu_sys"         greptime:"field;column:cpu_sys;type:float64"`
 }
+```
+
+Declare the table name as follows:
 
-func (Monitor) TableName() string {
-	return "monitor"
+```go
+func (CpuMetric) TableName() string {
+	return "cpu_metric"
 }
+```
 
-var monitor Monitor
-db.Raw("SELECT host, cpu, ts FROM users WHERE ts > ?", 1701360000000).Scan(&result)
+Use raw SQL to query data:
+
+```go
+var cpuMetric CpuMetric
+db.Raw("SELECT * FROM cpu_metric WHERE ts > ?", 1701360000000).Scan(&result)
```

Comment:
It is strange to leave a fixed ts here. Is it ok a simpler SQL? `SELECT * FROM cpu_metric`